data/json/professions.json

```diff
@@ -3417,5 +3417,368 @@
                 "panties"
             ]
         }
+    },
+	{
+
+            "type": "profession",
+            "ident": "elementary_student",
+            "name": "Elementary Student",
+            "description": "You're just a kid, and now the world has turned into something out of one of your bad dreams. The grown-ups you relied on are all dead--or undead--now. What are you going to do?",
+            "points": 0,
+            "items": {
+                "both": [                    
+                    "tshirt",
+                    "socks",
+                    "backpack",
+					"sneakers",
```

Comment:
Looks like some Tab characters have snuck into your changes here -- tell your Notepad++ that .json files should have spaces instead of tabs (four spaces is probably correct).